### PR TITLE
feat: Create minidump with Linux and Breakpad

### PR DESCRIPTION
**Pull request by bruno-garcia**

### CODE REVIEW

File: src/breakpad_wrapper.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+#elif defined(__linux__)
+
+bool callback(const MinidumpDescriptor &descriptor,
+              void *context,
+              bool succeeded) {
+    // if succeeded is true, descriptor.path() contains a path
+    // to the minidump file. Context is the context passed to
+    // the exception handler's constructor.
+    if (succeeded) {
+        SENTRY_PRINT_DEBUG_ARGS("Crashpad Minidump created at: %s\n", descriptor.path());
```
Comment (tonyo) [inline src/breakpad_wrapper.cpp]:
```suggestion
        SENTRY_PRINT_DEBUG_ARGS("Breakpad Minidump created at: %s\n", descriptor.path());
```

File: src/breakpad_wrapper.cpp [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    } else {
+        SENTRY_PRINT_ERROR("Crashpad minidump creation failed.");
```
Comment (tonyo) [inline src/breakpad_wrapper.cpp]:
```suggestion
        SENTRY_PRINT_ERROR("Breakpad minidump creation failed.");
```

File: src/breakpad_wrapper.cpp [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+}
+#endif
+
+ExceptionHandler *handler;
+
+int init(const sentry_options_t *options,
+         const char *minidump_url,
+         std::map<std::string, std::string> attachments) {
+
+    SENTRY_PRINT_DEBUG_ARGS("Initializing Crashpad with directory: %s\n", options->database_path);
```
Comment (tonyo) [inline src/breakpad_wrapper.cpp]:
```suggestion
    SENTRY_PRINT_DEBUG_ARGS("Initializing Breakpad with directory: %s\n", options->database_path);
```

Comment (mitsuhiko) [inline src/breakpad_wrapper.cpp]: should this not be `breakpad` here?

**Pull request merged**